helpers/parallel.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+function run_tasks_in_parallel() {
+  if ! command -v parallel >/dev/null; then
+    apt-get update >/dev/null
+    apt-get install --yes parallel >/dev/null
+  fi
+
+  TERM=linux
+  task_type=$1
+  echo "Running ${task_type} in Parallel."
+
+  shift
+  printf "\n\n"
+  _PARALLEL_OPTIONS="--halt-on-error now,fail=1 --will-cite"
+  if parallel ${_PARALLEL_OPTIONS} bash -c '"set -exo pipefail;"{}' ::: "$@"; then
+    echo "Running ${task_type} in Parallel Finished."
```

Comment:
printf vs echo is a little disorienting, looks like mostly it's for newlines... If so maybe just `echo -e`?